coalib/parsing/DefaultArgParser.py

```diff
@@ -177,6 +177,10 @@ def default_arg_parser(formatter_class=None):
         '-d', '--bear-dirs', type=PathArg, nargs='+', metavar='DIR',
         help='additional directories which may contain bears')
 
+    inputs_group.add_argument(
```

Comment:
This belongs before --bear-dirs , probably after --files .

Lets wait for more feedback on position

Reply:
Currently,
```
Inputs:
  -b, --bears           names of bears to use
  -f, --files           files that should be checked
  -i, --ignore          files that should be ignored
  --limit-files         filter the `--files` argument's matches further
  -d, --bear-dirs       additional directories which may contain bears
  -t, --tags            tags of sections to execute
```

I think `tags` should definitely be above `bear-dirs`.